components/butterfly/src/rumor.rs

```diff
@@ -231,10 +239,18 @@ mod storage {
         }
     }
 
+    impl<'a, C: ConstKeyRumor> IterableGuard<'a, RumorMap<C>> {
+        pub fn contains_id(&self, member_id: &str) -> bool {
+            self.get(C::const_key())
+                .map(|departures| departures.contains_key(member_id))
```

Comment:
Should this be called `rumors`, rather than `departures`, since this isn't _specifically_ about Departure rumors?

Reply:
Good catch! I initially specialized it for `Departures`, but now that it's using the `ConstKeyRumor` bound, that makes more sense. Fixed in https://github.com/habitat-sh/habitat/pull/6674/commits/55254f0c27ea87a7dd344e41b11b055a5ef158d5.

Reply:
I figured that's what happened 😄